guides/common/modules/proc_creating-a-webhook-template.adoc

```diff
@@ -1,16 +1,19 @@
 [id="creating-a-webhook-template_{context}"]
 = Creating a Webhook Template
 
+Webhook templates are used to generate the body of HTTP request to a configured target when a webhook is triggered.
 Use the following procedure to create a webhook template in the {ProjectWebUI}.
 
 .Procedure
-. In the {ProjectWebUI}, navigate to *Administer* *>* *Webhooks Templates*.
+. In the {ProjectWebUI}, navigate to *Administer* > *Webhook Templates*.
 . Click *Clone an existing template* or *Create Template*.
 . Enter a name for the template.
 . Use the editor to make changes to the template payload.
 +
 A webhook HTTP payload must be created using {Project} template syntax.
 The webhook template can use a special variable called `@object` that can represent the main object of the event.
+@object can be missing in case of certain events.
+You can determine what data are actually available with the @payload variable.
```

Comment:
IMHO both lines need some hightlighting for the vars. Maybe "`@object`"?

Reply:
You are right. Adding monospace.